cni-plugin/integration/calico/calico_test.go

```diff
@@ -0,0 +1,90 @@
+package calico_test
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"testing"
+)
+
+const (
+	ConfigDirectory = "/etc/cni/net.d"
+	CalicoConflist  = "10-calico.conflist"
+)
+
+// Given a directory, return a map of filename->struct{}
+func files(directory string) (map[string]struct{}, error) {
+	files, err := os.ReadDir(directory)
+	if err != nil {
+		return nil, err
+	}
+
+	fileNames := make(map[string]struct{}, len(files))
+	for _, f := range files {
+		fileNames[f.Name()] = struct{}{}
+	}
+
+	return fileNames, nil
+}
+
+func TestMain(m *testing.M) {
+	runTests := flag.Bool("integration-tests", false, "must be provided to run the integration tests")
+	flag.Parse()
+
+	if !*runTests {
+		fmt.Fprintln(os.Stderr, "integration tests not enabled: enable with -integration-tests")
+		os.Exit(0)
+	}
+
+	os.Exit(m.Run())
+}
+
+func TestLinkerdIsLastCNIPlugin(t *testing.T) {
+	t.Parallel()
```

Comment:
I believe this isn't needed given both tests (flannel and calico) are triggered through separate `go test` invocations.

Reply:
That's true and when we write more tests we can think about adding the line back.